### PR TITLE
Added admin page and GitHub CI

### PR DESCRIPTION
**Pull request by Misha999777**

### CODE REVIEW

File: src/Service/RestService.js [hunk 1]
```diff
@@ -44,6 +44,14 @@ export function login(loginRequest) {
     });
 }
 
+export function applyFeatures(featuresRequest) {
+    return request({
+        url: BASE_URL + "/features/apply",
```
Comment (BukerKo) [inline src/Service/RestService.js]: Maybe would be better to use `FEATURES_URL + "/apply"`?

File: src/Admin/Admin.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      this.setState({isLoading: false});
+      alert("Settings applied")
+    }).catch(error => {
+      this.setState({isLoading: false});
+      alert(error.message || 'Sorry! Something went wrong. Please try again!')
+    });
+  };
+
+  render() {
+    const listItems = this.state.data.map((item) =>
```
Comment (BukerKo) [inline src/Admin/Admin.jsx]: Why not using React Bootstrap components?

File: src/Admin/Admin.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    }).catch(error => {
+      this.setState({isLoading: false});
+      alert(error.message || 'Sorry! Something went wrong. Please try again!')
+    });
+  }
+
+  handleChange = (event) => {
+    const {id} = event.currentTarget;
+    this.getValue(id).enabled = event.target.checked;
+    this.forceUpdate();
```
Comment (BukerKo) [inline src/Admin/Admin.jsx]: It's not recommended to use `forceUpdate()` method. Probbly some other approach?

File: src/Admin/Admin.jsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import {Button, Container, Form} from "react-bootstrap";
+import {withRouter} from "react-router-dom";
+import {applyFeatures, getFeatures} from "../Service/RestService";
+import ReactLoading from "react-loading";
+import LoadingOverlay from "react-loading-overlay";
+
+import './Admin.css'
+
+class Admin extends React.Component {
```
Comment (BukerKo) [inline src/Admin/Admin.jsx]: Would be better to use `PureComponent` here

File: src/Student/CodeEditor/CodeEditor.jsx [hunk 5]
```diff
@@ -19,7 +19,7 @@ export default class CodeEditor extends React.PureComponent {
 
     componentDidMount() {
         getFeatures().then(result => {
-            this.setState({features: result["_embedded"].features})
+            this.setState({features: result})
```
Comment (Misha999777) [inline src/Student/CodeEditor/CodeEditor.jsx]: Fixed error loading features due to API changes

**Pull request merged**